modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -79,6 +79,12 @@ endif::[]
 allowUserDefinedWorkspaceNamespaces
 ----
 
+== Handling incompatible usernames or user ID-s
+
+{prod-short} server automatically checks usernames and\or ID-s for compatibility with Kubernetes objects naming convention before creating a {orch-namespace} from a template.
+In case of incompatible user name or ID, it will be reduced to nearest valid name by replacing groups of unsuitable symbols with an `-` symbol. To avoid collisions,
```

Comment:
```suggestion
Incompatible username or IDs are reduced to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. To avoid collisions,
```